src/azure-cli/azure/cli/command_modules/acs/tests/hybrid_2020_09_01/test_loadbalancer.py

```diff
@@ -13,6 +13,7 @@ class TestLoadBalancer(unittest.TestCase):
 
     def test_configure_load_balancer_profile(self):
         managed_outbound_ip_count = 5
+        managed_outbound_ipv6_count = 4
```

Comment:
This set of profile (hybrid_2020_09_01) is using a legacy API version **2020-11-01**, does ipv6 related properties introduced at that time? If not, I think we'd better not touch these test files.

Reply:
so 2020-11-01 does not have IPv6 changes but the test failed when I didn't add it because of the call to configure_load_balancer_profile ... should I just pass `None` instead?

Reply:
Yes, should ensure that not setting this field (pass `None`) wouldn't break the function.